flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliTableauResultView.java

```diff
@@ -153,7 +153,12 @@ private void printResults(AtomicInteger receivedRowCount, boolean isStreamingMod
 
             switch (result.getType()) {
                 case EMPTY:
-                    // do nothing
+                    try {
+                        Thread.sleep(1);
+                    } catch (InterruptedException e) {
+                        // get ctrl+c from terminal
+                        return;
```

Comment:
I think we can just ignore the exception here , do not need return (fall to break).